pkg/util/http/tags.go

```diff
@@ -0,0 +1,19 @@
+package http
```

Comment:
Why `util`? Where else will we use it? Can we move it for now as a private method to tags serializer?
I'd rather keep code encapsulated and in small scope unless it's necessary to share it between components.

Reply:
I wanted to store together serializer's code and code that generates regex. Also I guess with private method it will be harder to test?

Reply:
Ease of testing should NOT be a reason to break encapsulation.
I'm a fan of testing functionality, not functions. If possible, test it from a higher layer (test what is generated by configurer). This way you leave a room for refactoring of internal logic without changing tests.

Packages like utils/commons/shared and making everything public makes reasoning about code much harder. 

If you want to store the same code for generating regex and serializing the header, make a proper package. This is purely envoy thing. This should be somewhere around pkg/xds/envoy

Watch this https://www.youtube.com/watch?v=KrLFs6f2bOA
It's for Java (you are familiar with it as far as I remembed) for with more of a business apps context. Not saying we should do this 1:1 but this can be an inspiration how we can improve our code.

Reply:
Great! Thank you for detailed answer. I like the idea of creating proper package in `pkg/xds/envoy`, I'll do it in the further PR that adds regex. 

But for now let's keep method private and test it from higher level

Reply:
I like the iterative approach. Thanks.